airbyte-server/src/main/java/io/airbyte/server/handlers/DestinationDefinitionsHandler.java

```diff
@@ -90,6 +91,17 @@ static DestinationDefinitionRead buildDestinationDefinitionRead(StandardDestinat
     }
   }
 
+  @VisibleForTesting
+  static DestinationDefinitionReadWithJobInfo buildDestinationDefinitionReadWithJobInfo(StandardDestinationDefinition standardDestinationDefinition, SynchronousJobRead jobInfo) {
+    try {
+      return new DestinationDefinitionReadWithJobInfo()
+              .destinationDefinitionRead(buildDestinationDefinitionRead(standardDestinationDefinition))
```

Comment:
Doesn't `buildDestinationDefinitionRead` already throw a `KnownException` if it encounters an NPE? Is this try catch necessary?

Reply:
Good catch; I was able to inline the important part of this function, drop the try, and eliminate the function entirely (and its counterpart in source).  That cleans up the code.